### PR TITLE
10795 Updated link to My Business Registry Dashboard

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#10795

*Description of changes:*

- updated link from dashboard to My Business Registry dashboard
- changed partial routes to new config variables
- used enum for internal routes
- app version = 1.0.7

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -135,12 +136,21 @@ export default {
...
     appName: JSON.parse(packageJson).name,
     appVersion: JSON.parse(packageJson).version,
+    registryDashboard: `${process.env.BCROS_BASE_URL}dashboard`,
+    pprDashboard: `${process.env.PPR_URL}/dashboard`,
+    businessRegistryStaffDashboard: `${process.env.AUTH_URL}staff/dashboard/active`,
+    myBusinessRegistryDashboard: `${process.env.AUTH_URL}account/business`,
+    createBcRegistriesAccount: `${process.env.AUTH_URL}choose-authentication-method`,
+    signinBceidUrl: `${process.env.AUTH_URL}signin/bceid`,
+    signinBcscUrl: `${process.env.AUTH_URL}signin/bcsc`,
+    signinIdirUrl: `${process.env.AUTH_URL}signin/idir`,
```
Comment (severinbeauvais) [inline nuxt.config.js:153]: This centralizes all composite URLs (partial routes).

File: utils/user-product-helper.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export function getProductInfo (config, type: ProductCode): ProductI {
     case ProductCode.BUSINESS:
       return {
         image: 'BCRS_dashboard_thumbnail_image.jpg',
-        link: config?.authURL || 'link_not_configured',
+        link: config?.myBusinessRegistryDashboard || 'link_not_configured',
```
Comment (severinbeauvais) [inline utils/user-product-helper.ts:12]: This is the fix...

from `https://dev.bcregistry.ca/business/auth/`

to `https://dev.bcregistry.ca/business/auth/account/business`.

File: utils/user-product-helper.ts [hunk 3]
```diff
@@ -23,7 +23,7 @@ export function getProductInfo (config, type: ProductCode): ProductI {
     case ProductCode.PPR:
       return {
         image: 'PPR_dashboard_thumbnail_image.jpg',
-        link: config?.pprURL || 'link_not_configured',
```
Comment (severinbeauvais) [inline utils/user-product-helper.ts:26]: This old link already redirected to the PPR dashboard, but the new link is more direct.

File: utils/user-product-helper.ts (hunk 2 above)
Comment (cameron-freshworks) [reply utils/user-product-helper.ts:12]: Looks good 👍

**Review by cameron-freshworks — COMMENTED** [review]
Looks good! 👍

File: README.md [hunk 4]
```diff
@@ -22,6 +23,7 @@ $ npm run start
 
 # generate static project
 $ npm run generate
+$ npm run start
```
Comment (thorwolpert) [inline README.md:26]: start is for local preview only
generate is what creates the static site.
so maybe run goes under a different heading

**Review by doug-lovett — APPROVED** [review]

**Review by thorwolpert — APPROVED** [review]
LGTM - if you can, change that 1 thing in the README

File: README.md (hunk 4 above)
Comment (severinbeauvais) [reply README.md:28]: Does this look OK?

**Pull request merged**